### DISCUSSION TITLE
Including the video in just docs

### DISCUSSION THREAD

**Discussion by dewan-ahmed** [casey/just#1403] [General]
Hi @casey I'm wondering if you're planning to include the video from https://github.com/casey/just/issues/152 into just documentation. If yes, where and when is it expected to appear?

**Comment by casey** [comment]
Hey there, I decided that there were a bunch of changes that I wanted to make to the video before including it in the docs. (Speed, content, layout, formatting, colors, etc.) And that since I had a lot of specific requirements that would be hard to communicate and iterate over, I would do it myself, eventually. Sorry not to include the video you made, and I appreciate the effort.
